nixos/modules/config/shells-environment.nix

```diff
@@ -107,14 +107,28 @@ in
     };
 
     environment.shellAliases = mkOption {
-      default = {};
-      example = { ll = "ls -l"; };
+      default = {
+        ls = "ls --color=tty";
+        ll = "ls -l";
+        l  = "ls -alh";
+      };
+      example = { gst = "git status"; };
       description = ''
         An attribute set that maps aliases (the top level attribute names in
         this option) to command strings or directly to build outputs. The
         aliases are added to all users' shells.
+
+        <note><para>
+        The defaults are there only as a suggestion and disappear when any alias is set.
+        To include the default aliases in your configuration use:
+        <code>
+        environment.shellAliases = {
+          gst = "git status";
+        } // options.environment.shellAliases.default;
```

Comment:
Nitpick: would be better to use
```nix
environment.shellAliases = options.environment.shellAliases.default // {
  gst = "git status";
}
```

because otherwise the default aliases would override your customized ones.